### ISSUE TITLE
Implement MAX_SEQ_PER_DIM hyperparameter to address inference slowdowns

### ISSUE THREAD

**Issue by DavidUdell** [DavidUdell/sparse_circuit_discovery#67]
My current bottleneck is that there's a trade-off at ablation time between how _many_ datapoints I do inference on and how _long_ and how _much memory_ that inference takes. It's no good to just take a single datapoint, e.g., for each dim, because then that logit and those effects aren't representatively drawn from the dataset anymore.

Some feature dims are naturally sparse in this regard. Others are densely activated, and so are on two orders of magnitude more than the sparsely activated features are. The solution here is to implement a `MAX_SEQ_PER_DIM` hyperparameter, which will take just the most activating seq indices per dim for the densely activating features.
